automation/infrastructure/terraform/kcs/install_kubeflow/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kubeflow
+resources:
+  - bundle.yaml
```

Comment:
bundle.yaml and prometheus.yaml is missing

Reply:
Reworked with helm